spec/java/util/Optional.lsl

```diff
@@ -115,53 +154,58 @@ import "java/util/stream/_interfaces.lsl";
     // static methods
 
     // #problem
-    @Generic("T")
-    @GenericResult("T")
+    @Parameterized(["T"])
+    @ParameterizedResult(["T"])
     @static fun empty (): Optional  // #problem
     {
         result = _makeEmpty();
     }
 
 
-    @Generic("T")
-    @GenericResult("T")
+    @Parameterized(["T"])
+    @ParameterizedResult(["T"])
     @static fun of (obj: T): Optional
     {
-        required obj != null;
+        requires obj != null;
 
         if (obj == null)
-            self._throwNPE();
+            {_throwNPE();}
```

Comment:
Maybe brackets must be on two lines ? 
if (obj == null)
{
    throwNPE();
}
Like in another code style;